v2/internal/typescriptify/typescriptify.go

```diff
@@ -901,3 +909,13 @@ func differentNamespaces(namespace string, typeOf reflect.Type) bool {
 	}
 	return false
 }
+
+func typeClashWithReservedKeyword(input string) bool {
+	for _, v := range jsReservedKeywords {
```

Comment:
might be worth doing a `strings.TrimSpace(input)` here?

Reply:
agree, it won't hurt :)
will add it